s3fs/__init__.py,sha256=_6_Vs_vblhJSJw-62JVIBIM8kTKLhuwPFAIvt3hanls,160
s3fs/_version.py,sha256=6bdlgCVdZV2_pUJRiKG2KpKsYFMYrpbXLkUzz6oSKfc,500
s3fs/core.py,sha256=0PjXxF9sUBZ6pOHgdaWUAubL-JCenOLtxnmXdVtACuo,88183
s3fs/errors.py,sha256=uV3E1hLvL18ZrZEAXLaKoITtr8X6Cs2NcE97v2Um2WY,7779
s3fs/mapping.py,sha256=FoqEdMne7LXUL4HgPN4j6WsMsrwxpb53GynDhXs9VRI,237
s3fs/utils.py,sha256=gS_Dbm9tNdcJD-FOJ6PdjvHaAywDfA1a-k7Hn1YPdAQ,5246
s3fs-2024.6.0.dist-info/LICENSE.txt,sha256=3DWZ-ma8G8_6I2g4vi-04V-EzkgBJE7sk9kOUVz8WNE,1505
s3fs-2024.6.0.dist-info/METADATA,sha256=ak3nxskv2JwdKvuUkvQo-Tdd0rDRHPYkv_jmPLn1ybU,1599
s3fs-2024.6.0.dist-info/WHEEL,sha256=GJ7t_kWBFywbagK5eo9IoUwLW6oyOeTKmQ-9iHFVNxQ,92
s3fs-2024.6.0.dist-info/top_level.txt,sha256=Lf6EI3TdjlPu7TN-92IIY6c-GdrHnkrKGb1n0iwKooI,5
s3fs-2024.6.0.dist-info/RECORD,,
